.kitchen.yml.erb

```diff
@@ -0,0 +1,43 @@
+---
+# Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
+driver:
+  name: ec2
+  aws_ssh_key_id: <%= ENV['KEY_PAIR_NAME'] %>
+  region: us-east-1
+  security_group_ids: ["sg-a4fc5ec1"]
+  require_chef_omnibus: true
+  subnet_id: subnet-c96f61bd
```

Comment:
same is for nexus-rm. hardcoded subnet and SG

Reply:
Parameterized security group and subnet.